diff --git a/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-introspector.js b/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-introspector.js
index 24bfc6e..2ee0261 100644
--- a/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-introspector.js
+++ b/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-introspector.js
@@ -15,6 +15,7 @@ export class SqliteIntrospector {
             .selectFrom('sqlite_master')
             .where('type', 'in', ['table', 'view'])
             .where('name', 'not like', 'sqlite_%')
+            .where('name', 'not like', '_cf_%')
             .select('name')
             .orderBy('name')
             .$castTo();
